fastapi/models/fastapi_endpoint.py

```diff
@@ -18,6 +20,8 @@
 
 _logger = logging.getLogger(__name__)
 
+event_loop = asyncio.new_event_loop()
```

Comment:
Question here: do we want one event loop to handle multiple databases / endpoints? Or create one dynamically based on the DB(+endpoint)? Is there any benefit to do that?

Reply:
@sebalix What problem are you trying to solve here?   I don't think we can share the same event loop between requests. This is to ensure that the processing of requests made in parallel is isolated.  The only problem I can imagine if Odoo runs in multi-thread mode and not multi-process mode is that you end up with zombie threads for event loops that were created for deleted odoo threads.... In this case, we don't currently have a mechanism for managing the lifecycle of event loop threads (this isn't a problem in multi workers mode).